src/views/EmailAuth.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect, useState } from 'react';
+import { requestAPI, API_SEND_CODE, API_AUTH_EMAIL, API_MY_INFO } from '@utils/api';
+import { Button, Box, makeStyles, Container, Typography } from '@material-ui/core';
+import UosInput from '@components/UosInput';
+
+
+export default function EmailAuth() {
+    const userID = window.localStorage.getItem('userID');
+    const [userEmail,setUserEmail] = useState(null);
+    const [isCodeSend, setIsCodeSend] = useState(false);
+    const [code, setCode] = useState(null);
+
+    useEffect(async()=> {
+        try{
+            const res = await requestAPI(API_MY_INFO, userID);
```

Comment:
makeAPI함수는 함수를 반환합니다.
```suggestion
            const res = await requestAPI(API_MY_INFO(), userID);
```